cpp/src/arrow/util/bit_util_test.cc

```diff
@@ -849,11 +850,17 @@ TEST(BitUtil, RoundUp) {
   EXPECT_EQ(BitUtil::RoundUp(1, 1), 1);
   EXPECT_EQ(BitUtil::RoundUp(1, 2), 2);
   EXPECT_EQ(BitUtil::RoundUp(6, 2), 6);
+  EXPECT_EQ(BitUtil::RoundUp(0, 3), 0);
   EXPECT_EQ(BitUtil::RoundUp(7, 3), 9);
   EXPECT_EQ(BitUtil::RoundUp(9, 9), 9);
   EXPECT_EQ(BitUtil::RoundUp(10000000001, 10), 10000000010);
   EXPECT_EQ(BitUtil::RoundUp(10, 10000000000), 10000000000);
   EXPECT_EQ(BitUtil::RoundUp(100000000000, 10000000000), 100000000000);
+
+  // test overflow
```

Comment:
Can you add a similar test for `CeilDiv` as well?

Reply:
Sure. Similar tests have been added. Please take a look.